lib/resolver/homebrew-file.js

```diff
@@ -2,7 +2,7 @@ import relativeFile from './relative-file.js';
 
 export default function ({ path, target }) {
   return [
-    relativeFile({ path, target }),
-    '{BASE_URL}/Homebrew/homebrew-core/blob/master/Formula/' + target,
+    relativeFile({ path, target, baseUrl: 'https://github.com' }),
```

Comment:
Better now? Please don't merge this PR, because I would like to release a new version tonight. The middle mouse click was a heavily requested feature which I would like to release  independent from this PR.